packages/vitest/src/node/plugins/index.ts

```diff
@@ -113,6 +113,17 @@ export async function VitestPlugin(
             // https://github.com/vitejs/vite/pull/16453
             emptyOutDir: false,
           },
+          // @ts-expect-error environments only exists in Vite 6
```

Comment:
`@ts-expect-error` will break on v6 CI where this becomes valid, so we need double comment like this

https://github.com/vitest-dev/vitest/blob/ec1860b44766fb472ec8181769d9274aab636942/packages/vite-node/src/hmr/emitter.ts#L66-L68

Reply:
Ah. true